src/components/CompareResults/PaginatedCompareResults.tsx

```diff
@@ -205,7 +205,7 @@ function PaginatedCompareResults(props: PaginatedCompareResultsProps) {
 }
 
 interface PaginatedCompareResultsProps {
-  mode: 'light' | 'dark';
+  theme: string;
```

Comment:
If we already know that the only two possible themes are `light` or `dark` then I prefer `theme: 'light' | 'dark'`, but that's just a personal preference.

Reply:
Valid point.  I changed all to 'light' | 'dark'